website/blog/2022-03-30-version-068.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Announcing React Native 0.68
+authors: [kelset, shikaSD]
+tags: [announcement, release]
+---
+
+# Announcing 0.68
+
+Hello everyone! Today we are announcing the 0.68.0 release of React Native, with opt-in to the New React Native Architecture, bug fixes and more.
+
+### Sections
+
+- [Highlights of 0.68](/blog/2022/03/30/version-068#highlights-of-068)
+- [Opting in into the New Architecture](/blog/2022/03/30/version-068#opting-in-into-the-new-architecture)
+- [Website updates](/blog/2022/03/30/version-068#website-updates)
+- [Interested in helping React Native stabilise new releases?](/blog/2022/03/30/version-068#interested-in-helping-react-native-stabilize-new-releases)
+
+<!--truncate-->
+
+## Highlights of 0.68
+
+[Andrei Calazans](https://twitter.com/Andrei_Calazans) helped us selecting the most relevant changes that 0.68 brings along:
+
+### Introducing [Animated.Color](https://github.com/facebook/react-native/commit/ea90a76efef60df0f46d29228289f8fc1d26f350)
```

Comment:
```suggestion
### Introducing [Animated.Color](https://github.com/facebook/react-native/commit/ea90a76efef60df0f46d29228289f8fc1d26f350)
```

Currently, we miss the docs for this feature. Ideally the link should lead to the doc page instead of commit.

Reply:
yeah that's the reason why it links to the commit - it's also called out in the paragraph

Reply:
Hmm, can we consider not mentioning this feature then, until it has been fully developed and we can offer docs for it? 

It doesn't seems to be a very crucial API, which need a public expose ASAP.

Reply:
yeah I see your point, I lean towards your idea but I'll defer to @cortinico / @ShikaSD / @lunaleaps